src/IP.php

```diff
@@ -67,6 +69,11 @@ abstract class IP
      */
     protected $is_private;
 
+    /**
+     * @var array
+     */
+    protected static $private_ranges;
```

Comment:
I don't think this code should be in this PR.

Reply:
I agree, but it still needs to be in the code base.